src/main/webapp/app/assessment-shared/assessment-layout/assessment-layout.component.ts

```diff
@@ -0,0 +1,41 @@
+import { Component, Input, Output, EventEmitter } from '@angular/core';
+import { Complaint, ComplaintType } from 'app/entities/complaint';
+import { Result } from 'app/entities/result';
+import { Conflict } from 'app/modeling-assessment-editor/conflict.model';
+import { ComplaintResponse } from 'app/entities/complaint-response';
+
+/**
+ * The <jhi-assessment-layout> component provides the basic layout for an assessment page.
+ * It shows the header, alerts for complaints on top and the complaint form at the bottom of the page.
+ * The actual assessment needs to be inserted using content projection.
+ * Components using this component need to provide Inputs and handle Outputs. This component does not perform assessment logic.
+ */
+@Component({
+    selector: 'jhi-assessment-layout',
```

Comment:
This is totally unrelated to this PR. But I am curious why we actually name our components with `jhi-..`. Why not use `artemis-..` to not conflict with `jhi` components like `jhi-alert..`? Maybe we can start naming our newly added generic components this way?

Reply:
I agree, we have defined this in our angular.json file:
https://github.com/ls1intum/Artemis/blob/develop/angular.json#L57

This was probably just an oversight. It would be nice if we could define this **per feature module** (e.g. participation- or code-editor-). https://angular.io/guide/styleguide#style-02-07
It would be nice if we could move this discussion to a Github issue.

Reply:
I crated #843 for this discussion.